### DISCUSSION TITLE
「タグアイコン」の存在意義

### DISCUSSION THREAD

**Discussion by nkay0** [increments/qiita-discussions#353] [要望・アイデア・不具合]
最近[タグアイコンの変更申請](https://help.qiita.com/ja/articles/qiita-icon-edit-request)機能が追加されましたが、Qiitaにとって「タグアイコン」とはなんなのかが知りたいです。

> タグのアイコン画像を変更したい場合、変更の申請を送ることができます。
>
> タグアイコン変更申請ページで画像と申請理由を入力し送信します。
> 
> 送信から3営業日以内に運営が確認し、問題がないと判断したら新しいアイコンに変更されます。

いつどういうときにアイコンを変更すべきなのか？どのような「申請理由」が「問題がない」ものでどのような理由が問題あるものなのか？についての情報がなく、そもそもタグアイコンの存在意義（活用方法？）すら謎で、変更申請機能を使う必要性もわかりません。

私の知る限り、タグアイコンが表示されるのはタグページの左上のところだけで、あってもなくてもいいもののように思います。

また、現在および今後採用されるタグアイコンはすべて権利関係には問題がないのでしょうか。

**Comment by tanaka-asami415** [comment]
@nkay0 
タグアイコンは、文字列以外に絵や記号といったもので、ノンバーバルに認識していただける等の目的で使用しております😊
必要性を感じる方にご活用していただき、必要性を感じられない方に無理に変更申請を強要するものではありません。

Qiita利用規約にて、Qiitaを利用するに際して、著作権、特許権等の知的財産権を侵害する行為など、
法律違反行為は認めておりません。
https://qiita.com/terms

**Reply by nkay0** [reply]
例えば、ユーザーアイコン画像は、トップ画面でも個別記事画面でも記事検索画面でも、常にユーザー名の横に表示されています。
もしタグアイコン画像に存在意義を見出すなら、上記の画面でタグの横にもタグアイコン画像を置くとよいと思いますが、いかがでしょうか。

**Reply by tanaka-asami415** [reply]
@nkay0 
ありがとうございます🙏
そのページの情報の優先度で変えており、現在のデザインとなっております。
トップ画面や記事ページでは、タイトルや記事を書いたユーザー名を目立たせたく
タグアイコン画像をあえて表示していないのですが
タグランキングや、タグが重要なページには、タグアイコン画像を表示させています。
UIは都度アップデートしておりますので、今後変更する際は、今回のご意見も参考にさせていただきます。
